### PR TITLE
chore: add documentation to support new onnx node

### PR DESCRIPTION
**Pull request by jfrery**
closes https://github.com/zama-ai/concrete-ml-internal/issues/4358

### CODE REVIEW

File: docs/developer/support_new_onnx_node.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Supporting New ONNX Nodes in Concrete-ML
```
Comment (bcm-at-zama) [inline docs/developer/support_new_onnx_node.md:1]: it's no more Concrete-ML for a long time, but Concrete ML

Comment (jfrery) [reply docs/developer/support_new_onnx_node.md:1]: Fixed

**Review by RomanBredehoft — APPROVED** [review]
thanks !

**Review by kcelia — APPROVED** [review]
Thanks

**Pull request merged**